notebooks/deepmreye_example_usage.ipynb

```diff
@@ -9531,7 +9531,49 @@
     "<a name=\"download\"></a>\n",
     "## Download exemplary data\n",
     "\n",
-    "To run this notebook, you will need to download a sample dataset from Open Science Framework: [https://osf.io/mrhk9/](https://osf.io/mrhk9/). The dataset comprises fMRI data of 12 exemplary participants, who fixated at various locations on the screen. These locations are hard coded in the present script, but can be easily replaced e.g. by camera eye tracking labels.  "
+    "To run this notebook, you will need to download a sample dataset from Open Science Framework: [https://osf.io/mrhk9/](https://osf.io/mrhk9/). \n",
```

Comment:
To better view the notebook:

https://github.com/Remi-Gau/DeepMReye/blob/test_demo/notebooks/deepmreye_example_usage.ipynb